### ISSUE TITLE
Password requirements tooltips is ignored by screen readers

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#5223]
The password requirements tooltip on the ‘Create password’ view is ignored by screen readers on macOS. We need to expose the list of requirements and the status of each requirement (passing/failing) to screen readers.

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3499)
┆Reporter: Lesley Norton

**Comment by data-sync-user** [comment]
➤ Bianca Hidecuti commented:

Hi Lesley Norton,

We are already tracking this issue here: https://mozilla-hub.atlassian.net/browse/VPN-3387 ( https://mozilla-hub.atlassian.net/browse/VPN-3387|smart-link )

**Comment by data-sync-user** [comment]
➤ Matt Lichtenstein commented:

Keeping open as this was specifically not fixed in https://mozilla-hub.atlassian.net/browse/VPN-3387 ( https://mozilla-hub.atlassian.net/browse/VPN-3387|smart-link )

**Comment by data-sync-user** [comment]
➤ Raluca Monica Ilban commented:

Also, the Password field is not read by the screen reader on Windows 10, while using 2.19.0 (2.202310280712) build.
